docker/nginx/nginx.conf

```diff
@@ -41,7 +41,7 @@ http {
 
         location / {
             # try to serve file directly, fallback to app.php
-            try_files $uri /app.php$is_args$args;
+            try_files $uri /app_dev.php$is_args$args;
```

Comment:
These Docker files are here to have a dev environment with Docker, not for production use. So why not setting `app_dev` here? 
For production environment, you need to use the Docker image.